app/Resources/translations/estimate.fr.xlf~

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="fr" target-language="fr" datatype="plaintext" original="file.ext">
+    <header>
+      <tool tool-id="symfony" tool-name="Symfony"/>
+    </header>
+    <body>
+      <trans-unit id="d90c94be7e59472ace1efa907cfaf9c0" resname="estimate_title">
+        <source>estimate_title</source>
+        <target>privatisez un restaurant</target>
```

Comment:
I'm not skilled in grammar but do we say "privatisez" or "privatiser"? As the pronoun is missing I guess it would be "privatiser". But I might be wrong.